src/Prototype/Exe/Runtime.hs

```diff
@@ -115,13 +115,13 @@ instance S.DBStorage ExeAppM User.UserProfile where
       ML.localEnv (<> "Storage" <> "UserProfile" <> "UserCreate") $ do
         ML.info
           $  "Creating new user: "
-          <> (show . User._userCredsId $ User._userCreds newProfile)
+          <> (show . User._userCredsName $ User._userProfileCreds newProfile)
           <> "..."
         onUserIdExists newProfileId createNew existsErr
      where
       newProfileId = S.dbId newProfile
       createNew    = onUserNameExists
-        (newProfile ^. User.userProfileName)
+        (newProfile ^. User.userProfileCreds ^. User.userCredsName)
```

Comment:
```suggestion
        (newProfile ^. User.userProfileCreds . User.userCredsName)
```

Reply:
It seems it was already done. Maybe you commented on a specific commit ? (Although I can't seem to find it.)

Reply:
Ah no, it is specifically the `.`, ok, will do.